frontend/src/components/datasets/PublicDataset.tsx

```diff
@@ -13,20 +13,18 @@ import { useParams, useSearchParams } from "react-router-dom";
 import { RootState } from "../../types/data";
 import { useDispatch, useSelector } from "react-redux";
 import { fetchPublicFolderPath } from "../../actions/folder";
-
 import {
 	fetchPublicDatasetAbout,
 	fetchPublicFoldersFilesInDataset as fetchPublicFoldersFilesInDatasetAction,
 } from "../../actions/public_dataset";
-
 import { a11yProps, TabPanel } from "../tabs/TabComponent";
 import FilesTable from "../files/FilesTable";
 import { DisplayMetadata } from "../metadata/DisplayMetadata";
 import { DisplayListenerMetadata } from "../metadata/DisplayListenerMetadata";
 import { MainBreadcrumbs } from "../navigation/BreadCrumb";
 import {
 	deleteDatasetMetadata as deleteDatasetMetadataAction,
-	fetchMetadataDefinitions,
+	fetchDatasetMetadata, fetchMetadataDefinitions, fetchPublicMetadataDefinitions,
```

Comment:
I don't think you are using the "fetchPublicMetadataDefinitions"? Seems like it's still using the private route. Could you double check if the conflict fix has introduce some change. Thanks!

Reply:
I got this fixed and committed.